tools/authoring-decisions.adoc

```diff
@@ -511,9 +524,43 @@ We have the expression constraint which accepts a FEEL expression, the enumerati
 The last two create the equivalent FEEL expression as well.
 Depending on the base type, the enumeration and range will not be available.
 
+[id="included-models"]
 === C. Included models
 
-. TODO
+By default, there is no included model and you should see an empty state screen on the **Included models** tab.
+
+image::tools/authoringDecisions/included-models-tab-empty-state.png[Included models tab empty state]
+
+To include an initial model, click the **(1) Include model** button.
+Once the button was clicked a modal will appear.
+The modal allows you to include all **DMN** and **PMML** models from the current workspace.
+
+image::tools/authoringDecisions/included-models-tab-modal.png[Include model modal]
+
+**(1) all models** is the current workspace name indicator.
+The **all models** is the workspace name used only for this demo.
+You can choose any workspace name you need.
+
+**(2) Model** is a dropdown that will list all **DMN** and **PMML** models from the **all models** workspace.
+
+**(3) Name** is a namespace we give to the included model, which will be used for referencing included nodes and data types.
+
+NOTE: With DMN 1.5. we can keep this empty to use the <<import-to-default-namespace, default namespace>> feature.
+
+**(4) Include model** is the button for confirming the include operation.
+Upon confirmation, the selected model(s) will be displayed below the **Included models** tab as a card.
+
+image::tools/authoringDecisions/included-models-tab-cards.png[Included models cards]
+
+**(1)** indicates the number of included models in your model.
+The number **2** indicates two included models.
+
+**(2)** is a card displaying details of the included model.
+
+**(3)** is the namespace we gave to the included model.
+We can change the namespace **includedLogic** by double-clicking on this value and typing a new namespace value.
+
+**(4)** gives an option to remove the included model.
```

Comment:
It would be good to add the case where the user import a DMN where the included model is not present on the workspace.

Reply:
thank you, good suggestion, done